src/Control/Parallel.purs

```diff
@@ -48,3 +50,13 @@ parSequence_
   => t (m a)
   -> m Unit
 parSequence_ = parTraverse_ id
+
+parOneOf
+  :: forall a t m f
+   . Parallel f m
+  => Plus f
```

Comment:
Right, my point was just that _this_ function only takes advantage of Plus, and if anyone using it needs the additional laws, they can request an `Alternative` constraint. But that's also like saying functions should be using `Bind` and `Applicative` instead of `Monad`, which is silly.